person.rb

```diff
@@ -1,13 +1,24 @@
-class Person
+class Nameable
```

Comment:
- [ ] [OPTIONAL] Kindly use separate files for `Nameable`, `Decorator`, `CapitalizeDecorator`, and `TrimmerDecorator` and use the `require` or `require_relative` instead of plotting the `person` file with all classes.